source/minimessage.rst

```diff
@@ -90,43 +90,24 @@ MiniMessage exposes a simple API via the ``MiniMessage`` class.
 
 There are two different instances of that interface, ``get()`` and ``markdown()``, the latter one provides basic Markdown support, in addition to the MiniMessage support. More information can be found in the Markdown_ section.
 
-MiniMessage allows you to both serialize components into MiniMessage strings and to parse/deserialize MiniMessage strings into components.
-
-Placeholder
-^^^^^^^^^^^^^^^^^^^
-
-TODO write about placeholders and templates
+Additional customization of MiniMessage is possible via the Builder_.
 
-Template
-----------
+MiniMessage allows you to both serialize components into MiniMessage strings and to parse/deserialize MiniMessage strings into components.
 
 Format
 ^^^^^^^^^^^^^^^^^^^
 
-
 This library uses tags. Everything you do will be defined with tags. Tags have a start tag and an end tag (the ``<reset>`` tag is an exception here).
 Start tags are mandatory (obviously), end tags aren't.
 ``<yellow>Hello <blue>World<yellow>!`` and ``<yellow>Hello <blue>World</blue>!`` and even ``<yellow>Hello </yellow><blue>World</blue><yellow>!</yellow>`` all do the same.
 
-Some tags have inner tags. Those look like this: ``<tag:inner>stuff</tag>``. For example: ``<hover:show_text:"<red>test:TEST">TEST`` or ``<click:run_command:test>TEST``  
+Some tags have inner tags. Those look like this: ``<tag:inner>stuff</tag>``. For example: ``<hover:show_text:"<red>test:TEST">TEST`` or ``<click:run_command:test>TEST``
 As you can see, those sometimes contain components, sometimes just strings. Refer to the detailed docs below.
 
-Single (``'``) and double (``"``) quotes can be used interchangeably, but please stay consistent. 
-
-The components try to represent vanilla as closely as possible. 
-It might to helpful to use `the minecraft wiki <https://minecraft.gamepedia.com/Raw_JSON_text_format>`_ as a reference, especially for stuff like the actions and values of click and hover events. 
+Single (``'``) and double (``"``) quotes can be used interchangeably, but please stay consistent.
```

Comment:
If there is a real reason for consistency you should say so, and what type of consistency you mean(use them in pairs, never mix them in a single MiniMessage string, always use one and not the other in inner tags etc.)